exporters/otlp/common/src/test/java/io/opentelemetry/exporter/internal/otlp/traces/TraceRequestMarshalerTest.java

```diff
@@ -109,16 +127,25 @@ void toProtoResourceSpans() {
     assertThat(onlyResourceSpans.getScopeSpansCount()).isEqualTo(1);
     ScopeSpans instrumentationLibrarySpans = onlyResourceSpans.getScopeSpans(0);
     assertThat(instrumentationLibrarySpans.getSchemaUrl()).isEqualTo("http://url");
-    assertThat(instrumentationLibrarySpans.getScope())
+    InstrumentationScope scope = instrumentationLibrarySpans.getScope();
+    assertThat(scope.getName()).isEqualTo("testLib");
+    assertThat(scope.getVersion()).isEqualTo("1.0");
+    assertThat(scope.getAttributesCount()).isEqualTo(2);
+    List<KeyValue> attributes =
+        scope.getAttributesList().stream()
+            .sorted(Comparator.comparing(KeyValue::getKey))
+            .collect(Collectors.toList());
+    assertThat(attributes.get(0))
         .isEqualTo(
-            InstrumentationScope.newBuilder()
-                .setName("testLib")
-                .setVersion("1.0")
-                .addAttributes(
-                    KeyValue.newBuilder()
-                        .setKey("key")
-                        .setValue(AnyValue.newBuilder().setStringValue("value").build())
-                        .build())
```

Comment:
Now that there's more than one attribute, we can't rely on the order and this test fails, so it had to be reworked to sort.